.expeditor/verify.pipeline.yml

```diff
@@ -0,0 +1,20 @@
+expeditor:
+  defaults:
+    buildkite:
+      timeout_in_minutes: 5
+
+steps:
+  - label: "Generate Pipeline"
+    command:
+      - ./scripts/generate_verify_pipeline.sh
+    agents:
+      queue: 'TBD'
+    plugins:
+      docker#v3.0.1:
+        always-pull: true
+        user: "buildkite-agent"
+        group: "buildkite-agent"
+        image: "chefes/buildkite"
+    retry:
+      automatic:
+        limit: 1
```

Comment:
Is this one attempt total, or 1 retry (i.e. 2 attempts total)? 
Sometimes the docker pull doesn't work right, I'd suggest trying at least a couple times.

Reply:
I think it's one retry, but will verify. What do you think is a reasonable number?

Reply:
This is 1 retry, fwiw, so a max of 2 attempts.